unit10.js

```diff
@@ -0,0 +1,127 @@
+// Bai1 Phep nhan
+const calcMultipliers = n => {
+    for (let i=1; i<=10; i++) {
+        console.log(`${n} x ${i} = ${n*i}`)
+    }
+}
+
+// Bai2 So chan
+const getEvenNumber = n => {
+    if (n >= 1 && n <= 30) {
+        for (let i=1; i<=(n/2); i++) {
+            console.log(2*i)
+        }
+    }
+}
+
+// Bai3 Tinh tong tu 1 den n
+const total = n => {
+    let result = 0
+    if (n>=1 && n<=30) {
+        for (let i=1; i<=n; i++) {
```

Comment:
Viết như thế này thì sẽ dễ đọc hơn:
```suggestion
    let result = 0
    if (n < 1 || n > 30) return;
    
    for (let i = 1; i <= n; i++) {
```
Hạn chế `if else` và các block lồng nhau lại

Ngoài ra nhìn kỹ các dấu cách bên trên (6)